README.md

```diff
@@ -135,6 +135,20 @@ Please see the [scikits.odes
 documentation](https://scikits-odes.readthedocs.io/en/latest/installation.html) for more
 detailed installation instructions.
 
+### [scikit.fem](https://github.com/kinnala/scikit-fem)
```

Comment:
since `scikit.fem` is just a pip package, we should just add it to `setup.py` and `.requirements-docs.txt`, then we don't need separate instructions for installation (it will automatically install with `pip install .`) 
Then we also don't need separate options in travis

Reply:
`scikit.fem` requires python 3.6+, so it looks like we'll need to keep it as optional unless we drop 3.5?

Reply:
Ok, fair enough